src/templates/factory.index.js.template

```diff
@@ -2,7 +2,11 @@
 import <%= component %> from './<%= component %>';
 <% }); %>
 
-console.warn('Support for factories will be removed in v0.25, for details see https://github.com/react-bootstrap/react-bootstrap/issues/825');
+if (process.env.NODE_ENV !== 'production') {
+  if ((typeof console !== 'undefined') && (typeof console.warn === 'function')) {
```

Comment:
Nope. It has different api. `deprecationWarning(oldname, newname, link)` [deprecationWarning.js#L1](https://github.com/react-bootstrap/react-bootstrap/blob/3a254a14f2d8a083ee76559533b8be02f2258921/src/utils/deprecationWarning.js#L1)
This was my first thought too :smile: 

Because it is a single case I've decided to do it straightforward as in this PR.
Otherwise we should add another one `deprecating` method to `utils/deprecationWarning`.

I think we always can add another one `deprecating` method later,
when - and if - there will be more of such deprecations.